src/Modules/SimplCommerce.Module.ShoppingCart/Areas/ShoppingCart/Controllers/CartController.cs

```diff
@@ -107,6 +107,19 @@ public async Task<ActionResult> ApplyCoupon([FromBody] ApplyCouponForm model)
             return Json(validationResult);
         }
 
+        [HttpPost("cart/save-ordernote")]
+        public async Task<IActionResult> SaveOrderNote([FromBody] SaveOrderNote model)
+        {
+            var currentUser = await _workContext.GetCurrentUser();
+            
+            if (model.OrderNote != "")
```

Comment:
As the OrderNote can be empty, I think we should remove the if statement to allow the customer to delete added OrderNote. Btw, for string, string.IsNullOrWhiteSpace is better than != ""

Reply:
Thanks, I have done the Suggested Change